stable/concourse/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 3.7.4
+version: 3.8.4
```

Comment:
Ops, little detail - should be `3.8.0` instead of `3.8.4` when doing a minor on `3.7.x` 😁

Reply:
Oops, fixed!